framework/src/modules/http_api/controllers/voters.js

```diff
@@ -161,48 +177,12 @@ VotersController.getVoters = async function(context, next) {
 VotersController.getVotes = async function(context, next) {
 	const { params } = context.request.swagger;
 
-	let filters = {
-		address: params.address.value,
-		publicKey: params.publicKey.value,
-		secondPublicKey: params.secondPublicKey.value,
-		username: params.username.value,
-	};
-
-	let options = {
-		limit: params.limit.value,
-		offset: params.offset.value,
-		sort: params.sort.value,
-		extended: true,
-	};
-
-	// Remove filters with null values
-	filters = _.pickBy(filters, v => !(v === undefined || v === null));
-	options = _.pickBy(options, v => !(v === undefined || v === null));
+	const { filters, options } = getFilterAndOptionsFormParams(params);
```

Comment:
Same as above

Reply:
the underscore private function is only for class, it's fine for variables.

Reply:
oh I meant `_ getFilterAndOptionsFormParams`